src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,86 @@
+package duke;
+
+import java.io.FileOutputStream;
+import java.io.FileInputStream;
+import java.io.ObjectOutputStream;
+import java.io.ObjectInputStream;
+import java.io.FileNotFoundException;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/*
+ * Class used to store and load all your tasks from your local storage.
+ *
+ * @author Cui Shen Yi
+ * @version CS2103T AY22/23 Semester 1
+ */
+public class Storage {
+    private static final String STORAGE_PATH = "storage/storage.txt";
+
+    /**
+     * Method used to init the duke.Storage on a device, creating the directory and file if it doesn't exist.
```

Comment:
Not much to comment here, but perhaps you could consider being a little more specific about what initStorage() does in your description?